docs/infrastructure/eks-quickstart.md

```diff
@@ -149,9 +149,16 @@ Install the NGINX Ingress Controller:
     kubectl create namespace codacy
     helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
     helm repo update
-    helm upgrade --install --namespace codacy --version 4.3.0 nginx-ingress ingress-nginx/ingress-nginx -f values-nginx.yaml
+    helm upgrade --install --namespace codacy --version 4.8.3 nginx-ingress ingress-nginx/ingress-nginx -f values-nginx.yaml
     ```
 
+!!! warning
+  Fixing recently disclosed Nginx vulnerabilities ([CVE-2022-4886](https://github.com/kubernetes/ingress-nginx/issues/10570), [CVE-2023-5043](https://github.com/kubernetes/ingress-nginx/issues/10571), [CVE-2023-5044](https://github.com/kubernetes/ingress-nginx/issues/10572)) requires installing chart version 4.8.3, or later, and enabling `enableAnnotationValidations` as in [`values-nginx.yaml`](../values-files/values-nginx.yaml). This upgrade can also be applied to supported Codacy Self-Hosted versions before v13.0.
+
+  If a custom configuration distinct from [`values-nginx.yaml`](../values-files/values-nginx.yaml) was used instead, please access the aforementioned documentation on how to apply a suitable fix. Please consider components outside Codacy’s installation scope, relying on this Nginx ingress, could be negatively impacted by such chart upgrade.
+
+  According to Nginx [official documentation](https://github.com/kubernetes/ingress-nginx/tree/helm-chart-4.8.3#supported-versions-table) only Kubernetes versions 1.25 to 1.28 are supported. However, we have carefully tested our product on currently supported Kubernetes versions, using Nginx chart 4.8.3, and the installation is functional in accordance to our quality assurance standards.
+
```

Comment:
@tfrgoncalves This page doesn't seem the right place for this content, as this procedure applies to a new SH installation. For new SH installations, we'll now provide the updated `values-nginx.yaml`, instruct users to install nginx-ingress 4.8.3 (updated command), and there isn't much more to say.

Thus, I moved all this content to the [Self-hosted v13.0.0 release notes](https://github.com/codacy/docs/pull/1913/files#diff-0f076c5cec766edb397917269450edec8fe97ff047abc2b4d5bdba867b633128R28-R61). 

Let me know if you agree. 🙏 

```suggestion
```